### PR TITLE
PA-201 Extract "select standard" into its own page

### PR DESCRIPTION
**Pull request by benagain**

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/ApprovalsOuterApiClient.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    public class ApprovalsOuterApiClient
+    {
+        private readonly IApprovalsOuterApiHttpClient _client;
+
+        public ApprovalsOuterApiClient(IApprovalsOuterApiHttpClient client)
+        {
+            _client = client;
+        }
+
+        public async Task<CourseDeliveryModels> GetCourseDeliveryModels(
```
Comment (Pauljgraham) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/ApprovalsOuterApiClient.cs:16]: I think the name should be GetProviderCourseDeliveryModel

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Queries/GetCourseDeliveryModels/GetCourseDeliveryModelsQueryHandler.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+﻿using MediatR;
+using SFA.DAS.ProviderCommitments.Infrastructure;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Queries.GetCourseDeliveryModels
+{
+    public class GetCourseDeliveryModelsQueryHandler
```
Comment (Pauljgraham) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Queries/GetCourseDeliveryModels/GetCourseDeliveryModelsQueryHandler.cs:8]: Again the name should be GetProviderCourseDeliveryModels....

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Queries/GetCourseDeliveryModels/GetCourseDeliveryModelsQueryResponse.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using SFA.DAS.CommitmentsV2.Types;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ProviderCommitments.Queries.GetCourseDeliveryModels
+{
+    public class GetCourseDeliveryModelsQueryResponse
```
Comment (Pauljgraham) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Queries/GetCourseDeliveryModels/GetCourseDeliveryModelsQueryResponse.cs:6]: GetProviderCourseDeliveryModels

**Review by Pauljgraham — APPROVED** [review]
Some minor name adjustment for 

CourseDeliveryModel..... 
to
ProviderCourseDeliveryModel.....

because the delivery model is based on course and provider and the intent become a little clearer.

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/ApprovalsOuterApiClient.cs (hunk 1 above)
Comment (benagain) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/ApprovalsOuterApiClient.cs]: Changed 👍

**Pull request merged**